documentation/make.bat

```diff
@@ -1,35 +1,35 @@
-@ECHO OFF
```

Comment:
reviertan este cambio

Reply:
@fedevilensky en teoría ahi le arregle el end of line y se lo deje de vuelta en LF, fijate por las dudas